rules/windows/process_creation/proc_creation_win_msdt_diagcab.yml

```diff
@@ -15,8 +18,8 @@ detection:
         - OriginalFileName: 'msdt.exe'
     selection_cmd:
         CommandLine|contains|all:
-            - '/cab'
-            - '.diagcab'
```

Comment:
I've deleted this line as Elastic provides a similar rule to this but without looking for ".diagcab". What is your opinion?